tests/v3/index.test.ts

```diff
@@ -988,4 +988,95 @@ describe("OpenAPI3 features", () => {
     `)
     );
   });
+
+  it("paths include 'summary' and 'description'", () => {
+    const schema: OpenAPI3 = {
+      openapi: "3.0.1",
+      paths: {
+        "/": {
+          summary: "root summary",
+          description: "root description",
+          get: {
+            summary: "get summary",
+            description: "get description",
+            responses: {
+              "200": {
+                content: {
+                  "application/json": {
+                    schema: {
+                      type: "object",
+                      properties: {
+                        title: { type: "string" },
+                        body: { type: "string" },
+                      },
+                      required: ["title", "body"],
+                    },
+                  },
+                },
+              },
+            },
+          },
+        },
+      },
+      components: {
+        schemas: {
+          ErrorResponse: {
+            type: "object",
+            properties: {
+              error: { type: "string" },
+              message: { type: "string" },
+            },
+            required: ["error", "message"],
+          },
+          SearchResponse: {
+            type: "object",
+            properties: {
+              title: { type: "string" },
+              date: { type: "string" },
+            },
+            required: ["title", "date"],
+          },
+        },
+        responses: {
+          NotFound: {
+            content: {
+              "application/json": {
+                schema: { $ref: "#/components/schemas/ErrorResponse" },
+              },
+            },
+          },
+        },
+      },
```

Comment:
I think you don't use `components` at all and can just leave it out from the schema?

Reply:
You are correct! I was not paying attention.  I'll pare it down to the minimum necessary.

Reply:
[There we are](https://github.com/drwpow/openapi-typescript/pull/366/files/efed10f2a66e1cf1385de147ed01b784e8995b72..2ad93b18e57f6bb542240bbc5895fbfb70ddb3a9). Much nicer, I think